Gruntfile.js

```diff
@@ -181,6 +181,9 @@ module.exports = function(grunt) {
              ' && curl -X DELETE http://admin:pass@localhost:5984/medic-audit-test' +
              ' && kanso push http://admin:pass@localhost:5984/medic-test'
       },
+      test_integration_setup: {
+        cmd: 'npm install api/',
```

Comment:
Mine too.  I think backing up `/node_modules` before doing these tests, and re-instating the clean version afterwards is worthwhile.